<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWidget</class>
 <widget class="QWidget" name="UserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>406</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conactlist</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Buddys</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="TW_BUDDY">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="textbox_buddyname"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_buddynamesave">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>save</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BuddyAutoOperator">
           <property name="text">
            <string>AutoOperator</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BuddyAutoTalkpower">
           <property name="text">
            <string>AutoTalkpower</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BuddyAntiChannelBan">
           <property name="text">
            <string>AntiChannelBan</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Blocked</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="TW_BLOCKED">
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="textbox_blockname"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_blocknamesave">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BlockedAutoBan">
           <property name="text">
            <string>AutoBan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BlockedAutoKick">
           <property name="text">
            <string>AutoKick</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Names</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTableWidget" name="TW_NAMES">
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLineEdit" name="textbox_blockedname"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_blocknameNamesave">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>save</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BlockedNameAutoBan">
           <property name="text">
            <string>AutoBan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_BlockedNameAutoKick">
           <property name="text">
            <string>AutoKick</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Country</string>
    </attribute>
    <widget class="QTableWidget" name="TW_COUNTRY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>241</width>
       <height>341</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <column/>
    </widget>
    <widget class="QCheckBox" name="chk_BlockedCountryAutoBan">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>64</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AutoBan</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_BlockedCountryAutoKick">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>64</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AutoKick</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
